tests/node-sync/run_light.go

```diff
@@ -51,55 +51,36 @@ func RunLightNode(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 		return err
 	}
 
-	bridgeTotalCh := make(chan int)
-	sub, err := syncclient.Subscribe(ctx, testkit.BridgeTotalTopic, bridgeTotalCh)
-	if err != nil {
-		return err
-	}
-
-	var bridgeTotal int
-	select {
-	case err = <-sub.Done():
-		if err != nil {
-			return err
-		}
-	case bridgeTotal = <-bridgeTotalCh:
-		err = <-syncclient.MustBarrier(ctx, testkit.BridgeStartedState, bridgeTotal).C
-		if err != nil {
-			return err
-		}
-	}
-
-	bridgeCh := make(chan *testkit.BridgeNodeInfo)
-	sub, err = syncclient.Subscribe(ctx, testkit.BridgeNodeTopic, bridgeCh)
+	bridgeCh := make(chan *testkit.BridgeNodeInfo, runenv.IntParam("bridge")) //4
```

Comment:
4?

Reply:
you meant 42 ? 🤣

Reply:
Oh gosh, forgot to remove! Thanks for the nit! 💪